docs/appendices/serialisation.md

```diff
@@ -86,11 +86,11 @@ The custom deserialisation method is expected to modify the values of the object
+The programmer must write their custom serialisation and deserialisation code in such a way that it is aware of how many bytes are available in the byte arrays that are passed to the methods. If the objects are always of a fixed size then the functions can read and write that many bytes to the buffer. However, if the objects are of varying sizes (for example, if the object was a string), then the serialized representation must include information that the deserialisation code can use to ensure that it does not read beyond the end of the memory occupied by the object. The custom serialisation system does not provide a mechanism for doing this, so it is up to the program to choose a mechanism and implement it. In the case of a string, the serialisation format could consist of a 4-byte header that encodes the length of the string, followed by a string of the specified length. These additional four bytes must be included in the value returned by `_serialise_space()`. The deserialisation function would then start by reading the first four bytes of the array to obtain the size of the string and then read only that many bytes from the array.
 
 #### Classes With Multiple `Pointer` Fields
 
-If a class has more than one `Pointer` field then all of those fields must be handled by the custom serialisation and deserialisation methods for that class; there are not methods for each field. For example, if a class has three `Pointer` fields then the `_serialise_space()` method must return the total number of bytes required to serialise the objects from all three fields.
+If a class has more than one `Pointer` field then all of those fields must be handled by the custom serialisation and deserialisation methods for that class; there are not methods for each field. For example, if a class has three `Pointer` fields, then the `_serialise_space()` method must return the total number of bytes required to serialise the objects from all three fields.
```

Comment:
what changed here?

Reply:
I added a comma after "if a class has three `Pointer` fields", which I think improves the flow of reading

Reply:
I'm not sure that is gramatically correct. I'm not sure it isn't

Reply:
I have reverted this, since at least we know that the original version is correct